src/main/java/org/pesmypetcare/webservice/controller/petmanager/PetMedicationRestController.java

```diff
@@ -0,0 +1,136 @@
+package org.pesmypetcare.webservice.controller.petmanager;
+
+import org.pesmypetcare.webservice.entity.MedicationEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.pesmypetcare.webservice.service.MedicationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/medication")
+public class PetMedicationRestController {
+    @Autowired
+    private MedicationService medicationService;
+    /**
+     * Creates a Medication eaten by a pet on the database.
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param dateName Date + name of the Medication
+     * @param medication The Medication entity that contains the attributes of the Medication
+     */
+    @PostMapping("/{owner}/{petName}/{dateName}")
+    public void createMedication(@PathVariable String owner, @PathVariable String petName,
+                                 @PathVariable String dateName, @RequestBody MedicationEntity medication) {
+        medicationService.createMedication(owner, petName, dateName, medication);
+    }
+
+    /**
+     * Deletes all the medication with the specified owner and pet from certain date.
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param dateName date + name of the receipt of the medication
+     */
+    @DeleteMapping("/{owner}/{petName}/{dateName}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5348142)